core/tests/unit/ldml/keyboards/meson.build

```diff
@@ -0,0 +1,43 @@
+# Copyright:    © SIL International.
+# Description:  Cross platform build script to compile kmkbpldml API unit tests.
+# Create Date:  5 Aug 2022
+# Authors:      Marc Durdin
+#
+
+tests = [
+# disabling 000 and 002 until we have updates to core or to the keyboards so that they pass
+#  'k_000_null_keyboard',
+  'k_001_tiny',
+#  'k_002_null_invalid'
```

Comment:
I wouldn't expect kmldmlc to generate an invalid .kmx.  So perhaps null_invalid should be preserved for now? Or maybe a directory full of invalid .kmxes (with a starting population of 1)? Eventually I'd like to do some fuzz testing to validate the validator.